### PR TITLE
add button & dropdown examples

### PR DESCRIPTION
**Pull request by notwaldorf**
Land 
- [x] https://github.com/PolymerElements/paper-behaviors/pull/33
- [x] https://github.com/PolymerElements/paper-dropdown-menu/pull/57

Then add new examples for button and `paper-dropdown-menu`.

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -44,6 +47,15 @@
...
           <paper-checkbox name="food" value="cheese" required>I like cheese</paper-checkbox><br>
 
+          <paper-dropdown-menu label="Cars" name="cars" required>
+            <paper-menu class="dropdown-content">
+              <paper-item>Volvo</paper-item>
+              <paper-item>Saab</paper-item>
+              <paper-item>Fiat</paper-item>
+              <paper-item>Audi</paper-item>
+            </paper-menu>
+          </paper-dropdown-menu>
```
Comment (cdata) [inline demo/index.html]: :beers:

File: demo/index.html [hunk 2]
```diff
@@ -56,7 +68,7 @@
 
           <br><br>
 
-          <paper-button raised
+          <paper-button raised name="button" value="true"
```
Comment (cdata) [inline demo/index.html]: :beetle:

**Comment by cdata** [comment]
LGTM :+1:

**Comment by cdata** [comment]
@notwaldorf bump - since those two PRs are closed, is there an opportunity to land this one?

**Comment by notwaldorf** [comment]
Well, button was nuked. I should just update it to only add dropdown.

**Pull request merged**